src/core/manager.c

```diff
@@ -2010,6 +2010,16 @@ int manager_startup(Manager *m, FILE *serialization, FDSet *fds, const char *roo
         return 0;
 }
 
+static Transaction *transaction_abort_and_free(Transaction *tr) {
```

Comment:
Could you move this to transaction.c ?

Reply:
I intentionally haven't done that, because the function seems rather specific to me and it's only needed in `manager.c`. But one place is as good as the other...